spec/index.bs

```diff
@@ -1017,8 +1017,12 @@ its mapping.
      mapping.
 
 <script type=idl>
+callback BufferMapCallback = void (ArrayBuffer? mapping);
```

Comment:
I think the user would want to have more information here, let's say the `GPUBuffer` as another argument?